recipes/macse/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "macse" %}
+{% set version = "1.2" %}
+{% set sha256hash = "5545afd948cc4cd8af435ad9b29c78f77226b73c505b4a42cf938aad0d256803" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  # url: https://bioweb.supagro.inra.fr/{{ name }}/releases/{{ name }}_v{{ version }}.jar
+  url: https://bioweb.supagro.inra.fr/macse/releases/macse_v1.2.jar
+  sha256: {{ sha256hash }}
+
+build:
+  number: 0
+
+requirements:
+  run:
+    - openjdk >=1.5
+    - python
+
+test:
+  commands:
+    - macse -h
+    - macse --version
+
+about:
+  home: 'https://bioweb.supagro.inra.fr/macse/'
+  license: GPL
```

Comment:
GPL License requires the license file to be part of the distribution, thus the warning message you got. omicX shows the license for MACSE as `CeCILL version 2.1`, so I would suggest you to remove GPL and use `license: CeCILL 2.1` instead.